ksp/src/test/kotlin/com/tschuchort/compiletesting/AbstractSymbolProcessor.kt

```diff
@@ -0,0 +1,18 @@
+package com.tschuchort.compiletesting
+
+import org.jetbrains.kotlin.ksp.processing.CodeGenerator
+import org.jetbrains.kotlin.ksp.processing.Resolver
+import org.jetbrains.kotlin.ksp.processing.SymbolProcessor
+
+internal open class AbstractSymbolProcessor : SymbolProcessor {
```

Comment:
Shouldn't this be part of KSP?

Reply:
unfortunately it doesn't exist in the public API right now.
notice that this is only in tests